maps-compose/src/main/java/com/google/maps/android/compose/Polyline.kt

```diff
@@ -106,4 +213,4 @@ public fun Polyline(
             set(zIndex) { this.polyline.zIndex = it }
```

Comment:
I notice that all these `set()` calls should instead be `update()`, for reasons of performance & tidiness. It would be helpful to reflect this concern somewhere, but I'm not a maintainer. Could you create a separate issue for this? Or fold it into this one, but it would seem to mix different concerns.